extension/src/fileSystem/util.ts

```diff
@@ -1,6 +1,7 @@
 import { sep, parse } from 'path'
 
-export const getPathArray = (path: string): string[] => path.split(sep)
+export const getPathArray = (path: string): string[] =>
+  path.replace(/\\/g, '/').split('/')
```

Comment:
On further inspection, while this fixes image by step plots, it breaks regular image plots on windows due to the manually joined paths (using `sep`) are different from the paths inside the plots data (using `/` or `\`). I think adjusting how our `getType` works should fix the issue. 

Will get back to this tomorrow.

Reply:
Ended up adjusting the output data as a whole to make sure that plots data and path data matched up correctly.

Reply:
I think it would be safer to leave this function (revert the behaviour) and more `path.replace(/\\/g, '/').split('/')` into `standardisePath` only.

The function also needs a different name to help differentiate it from `standardizePath` here: https://github.com/iterative/vscode-dvc/blob/4cdac53936bf4555996d901f5fc56cf132e4afd9/extension/src/fileSystem/path.ts#L3

Reply:
[Q] How about `ensureOsFileSep` or something similar?